.github/workflows/_cleanup.yml

```diff
@@ -6,7 +6,33 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
-  remove-workload-and-cluster:
+  cleanup_registry:
+    name: Cleanup Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log into Azure
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - name: Set Azure Subscription
+        run: az account set --subscription ${{ secrets.ACC_LAB_SUBSCRIPTION_ID }}
+
+      - name: Log in to Azure Container Registry
+        run: az acr login --name ${{ secrets.AZURE_REGISTRY_NAME }}
+
+      - name: Cleanup Registry
+        run: |
+          # The filter looks for at least one digit as a proxy for auto
+          # generated tags
+          az acr run \
+            --cmd "acr purge --filter 'private/acc/samples/kafka/.*:.*' --ago 1d" \
+            --registry ${{ secrets.AZURE_REGISTRY_NAME }} \
+            /dev/null
```

Comment:
I think we may need to use https://learn.microsoft.com/en-us/azure/container-registry/container-registry-delete#delete-by-tag instead of the purge command.  Since we already know the tag from the pipeline run, hopefully we can swap it out.

Reply:
Thinking about this more, if there are 2 PRs that are working with multiple devs, and one of them is not cleaning their container to do additional debugging, purging on `.*:.*` can really trip them up.  We should really be deleting the specific container referenced by the tag used in this pipeline run.  If anyone causes a container to not be deleted, they're responsible for deleting when they're done.